src/files-card-body.tsx

```diff
@@ -48,6 +51,8 @@ function compare(sortBy: Sort): (a: FolderFileInfo, b: FolderFileInfo) => number
     // treat non-regular files and infos with missing 'size' field as having size of zero
     const size = (a: FolderFileInfo) => (a.type === "reg" && a.size) || 0;
     const mtime = (a: FolderFileInfo) => a.mtime || 0; // fallbak for missing .mtime field
+    // mask special bits when sorting
+    const perms = (a: FolderFileInfo) => a.mode ? (a.mode & (~(0b111 << 9))) : 0;
```

Comment:
This added line is not executed by any test.

Reply:
This is likely because the test never tests for the `0` case, which can be funnily enough easily done by in a test:
```
chmod 0 foo
```


```
a = { mode: 0 }
a.mode ? 'lol' : 'bar'
bar
```

Technically this is totally wrong... we probably want to check === undefined or null but eh

Reply:
actually, I dont know why this is marked as untested line?
There is `chmod 000 ...` in one testcase